src/scripts/components/stories/pilot-story/components/chapter-conclusion/chapter-conclusion.module.styl

```diff
@@ -1,14 +1,13 @@
 @require '../../../../../../variables.styl'
 
 .conclusion
-  height: 100vh
+  height: calc(100svh - 75px)
```

Comment:
What is that `-75px`? The search bar height? Maybe put it into a variable or add a comment.